src/components/listItemComponent/ListItemComponent.jsx

```diff
@@ -0,0 +1,34 @@
+import { useSelector } from "react-redux";
+import { selectRedditReturn } from '../reddit/RedditSlice'
+
+const ListItemComponent = (props) => {
+    const redditApiReturn = useSelector(selectRedditReturn);
+    console.log('redditApiReturn', redditApiReturn)
+
+    if(!redditApiReturn.data) {
+        console.log('api return is not true')
+        return <div>reddit is loading...</div>
+    } else 
+        return (
+            <div>
+                {
+                    redditApiReturn.data.children.map((each, index)=> {
+                        return (
```

Comment:
define all the data that your list item needs here (what would have been passed down as props) You're going to have to use a .map to get the all_awardings.  Then use the each variable instead of dotting down into the object each time. Whatever isn't used, just comment it out for now, but make sure it's there.

Reply:
ok I am taking a break for now but I will do that shortly